### DISCUSSION TITLE
preload collection when building docker image in milvus 2.0

### DISCUSSION THREAD

**Discussion by TepesDracula** [milvus-io/milvus#16763] [Q&A and General discussion]
Hey, 
In previous milvus version (1.1) there was a method mentioned in doc on how to load collection while docker image is built(preload_collection).
I can't find this in 2.0.0 documentation, is this feature removed?

**Comment by yhmo** [comment]
This is the documentation of how to load collection in v2.0:
https://milvus.io/docs/v2.0.x/load_collection.md
